yargs.js

```diff
@@ -661,6 +669,11 @@ function Yargs (processArgs, cwd, parentRequire) {
       if (~handlerKeys.indexOf(cmd) && cmd !== completionCommand) {
         setPlaceholderKeys(argv)
         return command.runCommand(cmd, self, parsed)
+      } else {
+        if (recommendCommands) {
+          const similiarCommand = didYouMean(cmd, handlerKeys)
+          console.log('Did you mean \'' + similiarCommand + '\'?')
```

Comment:
let's pull this into our translation files, and make sure we support multiple languages 👍 -- we'll need to start passing the `y18n` dependency into this module like we do for `usage`, etc.